multichain-node/entrypoint.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/bash
 
+echo "Sleep for 10 seconds so the master node has initialised"
+sleep 10
```

Comment:
Is dit altijd nodig? Of alleen maar als de master node en de 'gewone' node (min of meer) tegelijk worden opgestart? Dat kan niet in de `docker-compose.yml` worden opgegeven, wel?

Reply:
Yes, alleen in het geval van de docker-compose setup. Het zou met een env variabele kunnen, maar denk dat het niet zo erg is om even te wachten.

Reply:
Voor in de cloud zou het beter zijn om niet onnodig te wachten, daar wil je zo veel mogelijk uptime hebben. Kan helaas niks vinden dat dat eenvoudig kan in Docker-compose. Mijn voorkeur gaat er dan toch naar uit om een env var te gebruiken